### DISCUSSION TITLE
Injecting React app with content scripts

### DISCUSSION THREAD

**Discussion by thiagodesa26** [crxjs/chrome-extension-tools#230] [Q&A]
Hi folks,

Awesome work you're doing here.

I'm using the vite beta version of the plugin and have an use case where the react app needs to be injected to the pages.

This would need to be done via [content scripts](https://developer.chrome.com/docs/extensions/mv3/content_scripts/). I tried setting the html.js resultant from the build command as the content script in the manifest.json but it errors out when the script tries to load react's vendor js since it uses the import key. I've tried multiple workarounds with no luck.

Are you able to provide a simple small example of usage for the plugin + react + ts + injection via content script?

Tks!

**Comment by jacksteamdev** [comment]
@thiagodesa26 This is coming in #224

I'll release an article with detailed instructions.

**Reply by Clovis-Breh** [reply]
I was about to ask same question I have same use case where I need to inject react app in website.
Good thing I looked for it first 🙂

**Comment by derekzyl** [comment]
just curious how did you guys solve the challenge?
